frontend/src/app/trees/forests/tree-guidelines/wizard-view.component.html

```diff
@@ -0,0 +1,100 @@
+<section class="usa-section">
```

Comment:
My understanding was that we were not moving forward with the wizard view, is my understanding not correct?

Reply:
@lauraGgit We might want to use this for the actual permit application, but we'll remove it for now.

Reply:
@lpendem can you please link to the story that gets created for that?